multinode/config/config.go

```diff
@@ -26,6 +26,7 @@ type MultiNode struct {
 	FinalizedBlockPollInterval *config.Duration
 	EnforceRepeatableRead      *bool
 	DeathDeclarationDelay      *config.Duration
+	VerifyChainID              *bool
```

Comment:
Would it be better to have `skipChainIDVerification` rather than `VerifyChainID`. That way the check is enabled by default

Reply:
The defaults are set per chain and must be added explicitly since it's a pointer, so I think it's fine in this case.